packages/circuit-ui/components/NotificationFullscreen/NotificationFullscreen.spec.tsx

```diff
@@ -42,14 +44,20 @@ describe('NotificationFullscreen', () => {
         onClick: jest.fn(),
       },
     },
-  };
+  } as const;
 
   describe('styles', () => {
     it('should render with default styles', () => {
       const { container } = renderNotificationFullscreen(baseProps);
       expect(container).toMatchSnapshot();
     });
 
+    it('should render with an SVG', () => {
```

Comment:
Nit (because the inline SVGs are also SVGs)

```suggestion
    it('should render with an SVG component', () => {
```